config/jobs/kubernetes/sig-network/sig-network-misc.yaml

```diff
@@ -3,32 +3,44 @@ presubmits:
   - name: pull-kubernetes-e2e-gci-gce-ipvs
     always_run: false
     optional: true
-    skip_report: true
+    skip_report: false
     branches:
     - master
+    annotations:
+      testgrid-create-test-group: "true"
     labels:
       preset-service-account: "true"
       preset-k8s-ssh: "true"
-    annotations:
-      testgrid-create-test-group: 'true'
+      preset-bazel-scratch-dir: "true"
+      preset-bazel-remote-cache-enabled: "true"
+      preset-pull-kubernetes-e2e: "true"
+      preset-pull-kubernetes-e2e-gce: "true"
     spec:
       containers:
       - args:
-        - --timeout=170
-        - --bare
+        - --root=/go/src
+        - --repo=k8s.io/kubernetes=$(PULL_REFS)
+        - --repo=k8s.io/release
+        - --upload=gs://kubernetes-jenkins/pr-logs
+        - --timeout=105
         - --scenario=kubernetes_e2e
         - --
+        - --build=bazel
+        - --cluster=
         - --env=KUBE_PROXY_MODE=ipvs
-        - --extract=ci/latest
-        - --gcp-master-image=gci
+        - --extract=local
         - --gcp-node-image=ubuntu
         - --gcp-zone=us-west1-b
         - --ginkgo-parallel=30
         - --provider=gce
         - --stage=gs://kubernetes-release-pull/ci/pull-kubernetes-e2e-gci-gce-ipvs
-        - --test_args=--ginkgo.skip=\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\] --minStartupPods=8
-        - --timeout=150m
-        image: gcr.io/k8s-testimages/kubekins-e2e:v20200324-0897dba-master
+        - --test_args=--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\] --minStartupPods=8
```

Comment:
@saschagrunert @justaugustus 
do we need something for this `--stage=gs://kubernetes-release-pull/ci/pull-kubernetes-e2e-gci-gce-ipvs` ? 
cc: @dims

Reply:
nope, it's a new directory in an existing bucket.